### ISSUE TITLE
Linked LIst bug (removing last subscription) 

### ISSUE THREAD

**Issue by jaggernoth** [aaronpowell/Postman#7]
Hi
I found that the linked list implementation has a bug: basically when removing element it's dealing with 'first' but not with 'last' item so if we are removing the last item from the list it's not handled properly 
i've discovered it when i was unsubscribing/resubscribing same method few times to the same event.

Not sure if it's 100% correct but here's the really quick fix i made directly in js (it works for me now)

``` javascript
    LinkedList.prototype.remove = function(node) {
      if (!node) {
        return;
      }
      if (!node.prev) {
        this.first = node.next;
        if (this.first) {
          this.first.prev = null;
        }
      } else {
        node.prev.next = node.next;
      }
      if (!node.next){
          this.last = node.prev
          if(this.last){
              this.last.next = null
          }
      } else{
          node.next.prev = node.prev;

      }
      return this.length--;
    };

```

basically it's the same as with the 'first' but reverted;)
